sdk/keyvault/Azure.Security.KeyVault.Certificates/tests/CertificateClientLiveTests.cs

```diff
@@ -934,7 +934,15 @@ public async Task DownloadECDsaCertificateSignRemoteVerifyLocal([EnumValues] Cer
             CertificateOperation operation = await Client.StartCreateCertificateAsync(name, policy);
             RegisterForCleanup(name);
 
-            await operation.WaitForCompletionAsync(DefaultCertificateOperationPollingInterval, default);
+            try
+            {
+                await operation.WaitForCompletionAsync(DefaultCertificateOperationPollingInterval, default);
+            }
+            catch (InvalidOperationException ex) when (ex.Message.Contains("An internal error has occurred"))
```

Comment:
This is probably due to an issue with the Test Proxy. Can we file an issue with repro instructions in the tools repo?

Reply:
As discussed offline, this may not be a test proxy issue but I'll consider adding an inner exception to what's thrown so we have more details.